### PR TITLE
Store logs of CI to GS

### LINKED ISSUE #1207: Upload CI logs to GS bucket and to artifacts folder in CircleCI job
**Summary of the defect**
All the CI job test results are not being uploaded to the google storage or to Artifacts section. This doesn't allow developer to retriever the logs. Fix the CI to upload the logs to GS.

**How to reproduce (if possible)**
https://app.circleci.com/pipelines/github/hashgraph/hedera-services/12502/workflows/9906cfcc-fb0f-49b6-bcba-07f1b6309cd8/jobs/60369

**Service logs (if applicable)**
```
...
```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
**Related issue(s)**:
Closes #1207 

**Summary of the change**:
Currently logs of Continuous Integration are not uploaded to google storage. This doesn't allow developers to see the logs if there is any error.
Modified to store logs of `continuous-integration-gcp`  to GS.

One of the CI run with results folder link : https://hedera-hashgraph.slack.com/archives/CMD3V6ZC4/p1617074380015200

**NOTE:** The branch checked out in regression will be reverted before merge, once the [PR](https://github.com/swirlds/swirlds-platform-regression/pull/951/files) in regression is merged. 
Space changes are necessary as the path for results folder breaks if there is any space in the path.

### CODE REVIEW

File: .circleci/config.yml
Comment (ljianghedera) [inline .circleci/config.yml:2244]: Restore this line before merge.

**Review by ljianghedera — DISMISSED** [review]
LGTM.

**Review by qnswirlds — APPROVED** [review]
Regression PR is approved. After that merge, as @ljianghedera, you could remove the `git checkout 1207-M-store-logs-to-gs;`

Minor concern: I am seeing `Fees failed to initialize!` that failed the `HAPIClientValidator` from the run that you provided

File: .circleci/config.yml
Comment (Neeharika-Sompalli) [reply .circleci/config.yml:2244]: Thanks @ljianghedera . As I mentioned in the PR description will merge it once regression PR is merged tomorrow.

**Comment by Neeharika-Sompalli** [comment]
> Regression PR is approved. After that merge, as @ljianghedera, you could remove the `git checkout 1207-M-store-logs-to-gs;`
> 
> Minor concern: I am seeing `Fees failed to initialize!` that failed the `HAPIClientValidator` from the run that you provided

@qnswirlds  Ah yes.. Thanks ! Updated the results link in PR description to https://hedera-hashgraph.slack.com/archives/CMD3V6ZC4/p1617074380015200

File: .circleci/config.yml
Comment (Neeharika-Sompalli) [reply .circleci/config.yml]: @ljianghedera @qnswirlds Reverted the regression branch.

**Review by qnswirlds — DISMISSED** [review]

File: .circleci/config.yml
Comment (ljianghedera) [inline .circleci/config.yml:398]: nit: If CI_PARAMETERS is always empty here, then you can directly use `CI_PARAMETERS="$CIRCLE_USERNAME $CIRCLE_BUILD_URL"` to reduce future confusion.

**Review by ljianghedera — APPROVED** [review]
LGTM other than the minor nit.

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Pull request merged**
